### PR TITLE
GML-1837: consolidate docker compose files

### PR DESCRIPTION
**Pull request by billshitg**
Code change:
* Combine docker compose files
* Fixed wrong tg_documents path in ecc

Still have error starting up copilot. Don't merge.
Will change the dev tag back to latest for the containers before merge.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,12 +18,9 @@ services:
       USE_CYPHER: "true"
     volumes:
       - ./configs/:/code/configs
-      - ./common:/code/common
-    networks:
-      - copilot_local
 
   eventual-consistency-service:
-    image: tigergraphml/ecc:latest
+    image: tigergraphml/ecc:dev
```
Comment (RobRossmiller-TG) [inline docker-compose.yml:23]: I think we should keep this as latest. Dev/nightly should be opt-in for users

**Review by RobRossmiller-TG — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (billshitg) [reply docker-compose.yml:23]: Yeah, will change back to latest before PR

**Review by RobRossmiller-TG — APPROVED** [review]
LGTM once latest tag is replaced in compose file

**Review by luzhoutg — APPROVED** [review]
LGTM

**Pull request merged**